.doc_gen/metadata/lambda_metadata.yaml

```diff
@@ -234,6 +304,39 @@ lambda_Scenario_GettingStartedFunctions:
             - description: Create a function that runs the scenario.
               snippet_tags:
                 - python.example_code.lambda.Scenario_GettingStartedFunctions
+    Ruby:
+      versions:
+        - sdk_version: 3
+          github: ruby/example_code/lambda
+          sdkguide:
+          excerpts:
+            - description: Set up pre-requisite IAM permissions for a Lambda function capable of writing logs.
+              snippet_tags:
+                - ruby.example_code.lambda.setup_iam
+            - description: Define a &LAM; handler that increments a number provided as an invocation parameter.
+              snippet_tags:
+                - ruby.example_code.lambda.handler.increment
+            - description: Zip your &LAM; function into a deployment package.
+              snippet_tags:
+                - ruby.example_code.lambda.create_deployment_package
+            - description: Create a new &LAM; function.
+              snippet_tags:
+                - ruby.example_code.lambda.CreateFunction
+            - description: Invoke your &LAM; function with optional runtime parameters.
+              snippet_tags:
+                - ruby.example_code.lambda.Invoke
+            - description: Update your &LAM; function's configuration to inject a new environment variable.
+              snippet_tags:
+                - ruby.example_code.lambda.UpdateFunctionConfiguration
+            - description: Update your &LAM; function's code with a different deployment package containing different code.
+              snippet_tags:
+                - ruby.example_code.lambda.UpdateFunctionCode
```

Comment:
I'm on the fence here: technically, you skipped the second invocation of the updated function because of the way you've snipped this out of the single action examples (instead of using your scenario code where you *do* include the second invocation).
On the other hand, I think it reads well in the output the way you have it here.
I'll leave it up to you whether you want to change it or not. 😊

Reply:
I'll keep it! I feel good about it